src/config/dynamic-theme-fixes.config

```diff
@@ -7557,6 +7557,23 @@ INVERT
 .serviceLogoImg
 .serviceIcon
 
+===============================
+
+kraken.com
+
+CSS
+header > nav > ul > li > a::after, ._3ku3YkWZbOVnguKutXHfK- {
+    background-color: var(--darkreader-neutral-text) !important;
+}
+a.has-text-primary, div.has-text-primary, .kraken-cta, label,
+a.nav-create, a.nav-create-account, a.nav-login {
+    color: var(--darkreader-neutral-text) !important;
+    background-color: var(--darkreader-neutral-background) !important;
+}
```

Comment:
I'm not able to find any of these selectors, are they on the account dashboard of some sort?

Reply:
Yeah, they are in a couple of different places

- https://support.kraken.com/hc/en-us
**a.nav-create, a.nav-create-account, a.nav-login**

This is without the fix

![image](https://user-images.githubusercontent.com/26639800/140543287-020c5c7c-6deb-46cc-bdde-7724bdf6e9bb.png)

This is with the fix

![image](https://user-images.githubusercontent.com/26639800/140542513-4348936a-df41-4a53-bb7c-fa19fad13951.png)

- https://www.kraken.com/en-us/institutions
**.kraken-cta** 

This is without the fix

![image](https://user-images.githubusercontent.com/26639800/140542240-15769314-f7cb-497a-a898-479b97a37d06.png)

With the fix

![image](https://user-images.githubusercontent.com/26639800/140542356-633b6a46-947f-45c0-be9f-447abaf9ab49.png)

- https://www.kraken.com/prices?quote=USD
**label**

This is without the fix (there are also other labels)

![image](https://user-images.githubusercontent.com/26639800/140543955-74647a13-9a4a-43eb-ae0a-7f73b7a57b6a.png)

This is with the fix

![image](https://user-images.githubusercontent.com/26639800/140544214-5ab61b73-a416-4b23-b87b-c4c706d23164.png)

- https://futures.kraken.com/

a.has-text-primary, div.has-text-primary

This is without the fix
![image](https://user-images.githubusercontent.com/26639800/140544585-b03a88b5-9300-4718-bc2d-fde90b0b48f0.png)

This is with the fix

![image](https://user-images.githubusercontent.com/26639800/140545000-b54cbc95-2bc4-4ee1-a9e9-49f3401eee96.png)

Probably there are other fixes inside the account dashboard but I don't have an account so... I focused on the problems that **erwangse** pointed out in the issue 🐞

Reply:
>     * https://support.kraken.com/hc/en-us
>       **a.nav-create, a.nav-create-account, a.nav-login**

Ehm, this seems like a cosmetic change, as the page without Dark Reader also indicates that those 2 buttons don't have a background-color.



>     * https://www.kraken.com/en-us/institutions
>       **.kraken-cta**

ditto as above.



> a.has-text-primary, div.has-text-primary

ditto as above.

`label` is also questionable for me, but not as questionable as the other ones.

> Probably there are other fixes inside the account dashboard but I don't have an account so... I focused on the problems that **erwangse** pointed out in the issue lady_beetle

Only a few issues that **erwangse** pointed out seems relevant to be fixed as others are just _meh_.

Reply:
Yeah, it's sometimes hard to read purple text on black background in that page

Okay, I'll make these changes!